goci-interfaces/goci-ui/src/main/resources/static/js/helpers/utils-helper.js

```diff
@@ -1,3 +1,22 @@
+/*
+The following variables are used by all dedicated pages:
+ */
+var global_fl;
+var global_raw;
+
+// This is the global variable storing solr fields:
+global_fl = 'accessionId,ancestralGroups,ancestryLinks,associationCount,association_rsId,authorAscii_s,author_s,authorsList,' +
+    'betaDirection,betaNum,betaUnit,catalogPublishDate,chromLocation,chromosomeName,chromosomePosition,context,' +
+    'countriesOfRecruitment,currentSnp,efoLink,ensemblMappedGenes,fullPvalueSet,genotypingTechnologies,id,initialSampleDescription,' +
+    'label,labelda,mappedLabel,mappedUri,merged,multiSnpHaplotype,numberOfIndividuals,orPerCopyNum,orcid_s,pValueExponent,' +
+    'pValueMantissa,parent,positionLinks,publication,publicationDate,publicationLink,pubmedId,qualifier,range,region,' +
+    'replicateSampleDescription,reportedGene,resourcename,riskFrequency,rsId,shortForm,snpInteraction,strongestAllele,studyId,' +
+    'synonym,title,traitName,traitName_s,traitUri';
+global_raw = 'fq:resourcename:association or resourcename:study';
+
+// Number of rows displayed on the datatables
+pageRowLimit = 5;
```

Comment:
You are right. These variables were not the same. Please let me know if you experience any problems. However, having these variables in ALL dedicated pages is the most utterly stupid thing I have experienced since I work with code. 


"Also, this kind of clean-up work should not be added a feature/bug fix branch since it conflates what is being addressed in the branch similar to the need to address each ticket with it's own branch."
Disagree. I wanted to use the association count solr field to the returned data. Should I add that field to all lists on all pages? If something, that IS bad practice.

Reply:
So far I haven't seen any obvious issues. Just asking whether the lists were compared across all pages to make sure they are the superset since they're probably not all the same list on all pages.

Yes, this repetition of the variable on all pages is not ideal. I don't know who allowed this to be used in the production code.